tutorials/Tutorial6_Better_Retrieval_via_Embedding_Retrieval.py

```diff
@@ -46,20 +46,14 @@ def tutorial6_better_retrieval_via_dpr():
     document_store.write_documents(docs)
 
     ### Retriever
-    retriever = DensePassageRetriever(
+    retriever = EmbeddingRetriever(
         document_store=document_store,
-        query_embedding_model="facebook/dpr-question_encoder-single-nq-base",
-        passage_embedding_model="facebook/dpr-ctx_encoder-single-nq-base",
-        max_seq_len_query=64,
-        max_seq_len_passage=256,
-        batch_size=2,
-        use_gpu=True,
-        embed_title=True,
-        use_fast_tokenizers=True,
+        embedding_model="sentence-transformers/multi-qa-mpnet-base-dot-v1",
+        model_format="sentence_transformers",
     )
 
     # Important:
-    # Now that after we have the DPR initialized, we need to call update_embeddings() to iterate over all
+    # Now that after we have the retriever initialized, we need to call update_embeddings() to iterate over all
```

Comment:
Update to "Now that we initialized the Retriever,"  to match `.md` file

Reply:
Good catch, thank you!